src/shared/components/Checkbox/Checkbox.tsx

```diff
@@ -37,48 +47,52 @@ export const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
     const isSelected = !!value
     const [isFocused, setIsFocused] = useState(false)
 
-    const onChangeHandler = (e: React.FocusEvent<HTMLInputElement>) => {
+    const handleChange = (e: React.FocusEvent<HTMLInputElement>) => {
       if (!disabled && onChange) {
         onChange(!value)
       }
     }
-    const onFocusHandler = (e: React.FocusEvent<HTMLInputElement>) => {
+    const handleFocus = (e: React.FocusEvent<HTMLInputElement>) => {
       if (!disabled) {
         setIsFocused(true)
         if (onFocus) {
           onFocus(e)
         }
       }
     }
-    const onBlurHandler = (e: React.FocusEvent<HTMLInputElement>) => {
+    const handleBlur = (e: React.FocusEvent<HTMLInputElement>) => {
       if (!disabled) {
         setIsFocused(false)
         if (onBlur) {
           onBlur(e)
         }
       }
     }
+
     return (
-      <CheckboxLabel disabled={disabled}>
-        <Container selected={value} disabled={disabled} isFocused={isFocused} error={error}>
-          <InnerContainer selected={value} disabled={disabled} error={error} isFocused={isFocused}>
-            <Input
-              name={name}
-              ref={ref}
-              type="checkbox"
-              data-multiple="false"
-              checked={isSelected}
-              disabled={disabled}
-              onChange={onChangeHandler}
-              onFocus={onFocusHandler}
-              onBlur={onBlurHandler}
-              {...props}
-            />
-            <Checkmark>{isSelected ? <Icon name={isIndeterminate ? 'dash' : 'check'} /> : null}</Checkmark>
-          </InnerContainer>
-        </Container>
-        {label && <LabelText variant="subtitle2">{label}</LabelText>}
-      </CheckboxLabel>
+      <>
```

Comment:
I don't think we should display 2 separate elements here, this may have unexpected effects on the layout. In other components I think we just return the helper text as part of the label? There's also some unexpected spacing between the checkbox and the helper because of this I think.